config.yaml

```diff
@@ -0,0 +1,30 @@
+options:
+  cwrbox_image:
+    description: |
+      URL from which to retrieve the cwrbox LXD image.
+      The image can also be provided as a Juju resource, with the
+      resource being used in favor of this URL if available.
+      The signature will be fetched as this URL with '.sig' added.
+    type: string
+    default: https://s3.amazonaws.com/jujubigdata/cwrbox/cwrbox.tar.gz
+  cwrbox_keys:
+    description: |
+      List of keys trusted to sign the cwrbox LXD image.
+      Keys can be either full keys, or full fingerprints (without spaces).
+      Key IDs (short nor long) cannot be used.
+    type: string
+    default: |
+      - 5762CA76ABD635A79185394EDB6E071B449C78B1  # johnsca@apache.org
+  install_sources:
+    description: PPAs from which to install LXD and Juju
+    type: string
+    default: |
```

Comment:
i have a dumb question. Why use the apt packages over the snaps?  It seems like a lot of tooling isn't going to be maintained in debs anymore...  I cite:

- charm-tools
- conjure-up

as two candidates in question. Are we signing up for pain later not integrating with snaps out of the gate?